brainstormforce/astra
# AST-2493 - List Item is overlapping the column border area while using spectra container

### Description

List Item is overlapping the column border area while using spectra container
JIRA - https://brainstormforce.atlassian.net/browse/AST-2493
### Screenshots

https://share.bsf.io/v1uQdkRb
### Types of changes


- Fix: List Item is overlapping the column border area while using spectra container
- Fix: Paragraph top margin was removed earlier but certain cases require it.



### How has this been tested?

- Take Spectra container layout.
- Set to 3 columns.
- Add border.
- Check frontend, list items should be aligned within the container.
### Checklist:
